src/environments/classic_control/mountain_car.jl

```diff
@@ -102,11 +103,19 @@ function RLBase.reset!(env::MountainCarEnv{A,T}) where {A,T}
     nothing
 end
 
-(env::MountainCarEnv{<:ContinuousSpace})(a) = _interact!(env, min(max(a, -1, 1)))
+function (env::MountainCarEnv{<:ContinuousSpace})(a::AbstractFloat)
+    @assert a in env.action_space
+    env.action = a
```

Comment:
If I understand it correctly, `env.action` is not used anywhere? (Needs confirm from @jbrea  ) If so, we can just remove that field from `env` and delete the assignment here.

Reply:
Yeah, it does not seem used but I thought that it should be set since the field was there. It can be useful for an end-user, but also something they can manage themselves.